packages/node-sdk/src/client.ts

```diff
@@ -657,11 +655,10 @@ export class BucketClient {
     await this._config.batchBuffer.flush();
   }
 
-  private _getFeatures({
-    enableTracking = true,
-    ...context
-  }: ContextWithTracking): Record<string, RawFeature> {
-    void this.syncContext({ enableTracking, ...context });
+  private _getFeatures(
+    options: ContextWithTracking,
+  ): Record<string, RawFeature> {
+    void this.syncContext(options);
```

Comment:
Again, assuming that enableTracking is not undefined. So i strongly recommend an assert in sync context (for future developers not to mess up)